test/functional/api/v2/template_controller_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_plugin_helper'
+
+class Api::V2::TemplateControllerTest < ActionController::TestCase
+
+  test "should import from git" do
+    post :import, { 'repo': File.expand_path(File.join('..', '..', '..', '..', '..'), __FILE__) + '/test/templates/core'}
```

Comment:
Why not using `ForemanTemplates::Engine.root + '/test/templates/core'`, I think that should be equivalent but less prone to errors.